vaccine_feed_ingest/runners/al/arcgis/normalize.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+# isort: skip_file
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from datetime import datetime
+from typing import Optional, List
+
+from vaccine_feed_ingest_schema import location
+
+from vaccine_feed_ingest.utils.validation import BOUNDING_BOX
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("al/arcgis/normalize.py")
+
+STATE = "AL"
+SOURCE_NAME = "al_arcgis"
+FETCHED_FROM_URI = "https://alpublichealth.maps.arcgis.com/apps/opsdashboard/index.html#/2b4627aa70c5450791a7cf439ed047ec"
+
+
+def _get_id(
+    data_id: str,
+    layer_id: str,
+) -> str:
+    return f"al_arcgis:{layer_id}_0_{data_id}"
```

Comment:
FYI - #362 is adding the layer id + data id to the data ([see here](https://github.com/CAVaccineInventory/vaccine-feed-ingest/pull/362/files#diff-93dd09409afd3c7f1bb5917b84055e4e2f9860ca4e3ca602d55573606dc57e48)). When that lands... this method might be able to use it!

Reply:
That's great! I'll watch that, and add it to al/arcgis when it lands.

Thanks for all the feedback 🙂